### PR TITLE
buildpacks/nix: 2.3 -> 2.3.9

### PR DESCRIPTION
**Pull request by FRidh**
Update to latest version.

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -28,6 +28,7 @@ New features
 - Add support for Python 3.8 in :pr:`840` by :user:`minrk`.
 - Add Hydroshare as content provider in :pr:`800` by :user:`sblack-usu`.
 - Update to Jupyter Notebook 6 and Lab 1.2 in :pr:`839` by :user:`minrk`.
+- Update Nix from 2.3 to 2.3.9 in :pr:`991` by :user:`FRidh`.
```
Comment (FRidh) [inline docs/source/changelog.rst:31]: wrong release

Comment (betatim) [reply docs/source/changelog.rst:31]: Like in your other PR: I don't quite understand why you added this comment to the change log for an already released version.

**Comment by FRidh** [comment]
Changelog removed.

**Comment by FRidh** [comment]
Figshare failure is unrelated.

**Comment by FRidh** [comment]
As far as I can tell this is ready to go.

**Comment by betatim** [comment]
Thanks for this version bump and sorry about being slow getting this merged.

**Pull request merged**